docs/hadoop/hdfs_service.md

```diff
@@ -67,10 +68,32 @@ which supports a comma-delimited list of directories to configure multiple disks
 ### Data Node <a name="Data_Node"></a>
 
 * Configuration Data: Its path is defined by *hadoop-data-node-configuration* configuration map.
-* Data Storage: If  *${hadoop-data-node.storage_path}* specified, blocks are stored in these paths, otherwise in the *hdfs/data* directory under the storage path.
+* Data Storage: If  *${hadoop-data-node.storage_path}* specified, blocks are stored in these paths, 
+                    otherwise in the *hdfs/data* directory under common storage path.
 * Host Configuration: Its path is defined by *host-configuration* configuration map.
 * Temp Data: It is in the *hadooptmp/datanode* directory under the storage path.
 
+## Data Migration <a name="Data_Migration"></a>
```

Comment:
Could this extend to also support other scenario: user hdfs is full, vm mount new disk, keep previous datapath and add new disk to existing HDFS. i remember currently support multipath config for hdfs

Reply:
added